src/main/java/team4/footwithme/stadium/service/StadiumServiceImpl.java

```diff
@@ -85,6 +92,8 @@ public StadiumDetailResponse updateStadium(StadiumUpdateServiceRequest request,
     public void deleteStadium(Long memberId, Long stadiumId) {
         Stadium stadium = validateStadiumOwnership(memberId, stadiumId);
         stadium.deleteStadium(memberId);
+        List<Court> courts = courtRepository.findActiveByStadiumId(stadiumId);
+        if (!courts.isEmpty()) courtRepository.deleteAll(courts);
```

Comment:
```suggestion
        if (!courts.isEmpty()) {
            courtRepository.deleteAll(courts);
        }
```
전 한줄이라도 이렇게 하는걸 좋아하긴합니다 ㅎㅎ